version.bzl

```diff
@@ -17,7 +17,7 @@
 # against.
 # This version should be updated together with the version of Bazel
 # in .bazelversion.
-BAZEL_VERSION = "5.2.0"
```

Comment:
I guess not in this PR, but we really should address that TODO down in line 28, or we'll very soon break rules_python for Bazel 5.x users by accident.